Data/member.json

```diff
@@ -413,6 +413,13 @@
             "description": "Bangladeshi | Studying Computer Science and Engineering at BRAC University, Dhaka | Webmaster, BRACU IEEE SB | Competitive Programmer |",
             "githuburl": "https://github.com/ZuhairHossain/",
             "linkedinurl": "https://www.linkedin.com/in/syed-zuhair-hossain/"
+        },{
```

Comment:
Please can you move the brace after the comma to the next line @johndao1005 ? Thank you.

Reply:
opps my bad. i will get it fix